MacDown/Code/Preferences/MPTerminalPreferencesViewController.m

```diff
@@ -0,0 +1,226 @@
+                                encoding:NSUTF8StringEncoding];
+            output = [output stringByTrimmingCharactersInSet:
+                      [NSCharacterSet whitespaceAndNewlineCharacterSet]];
+            
+            NSString *shellUtilityPath = [output stringByAppendingString:@"/bin/macdown"];
+            
+            // If MacDown was installed from the Homebrew cask then the macdown binary should exist at this path
+            if ([[NSFileManager defaultManager] fileExistsAtPath:shellUtilityPath])
+            {
+                NSURL *shellUtilityUrl = [NSURL fileURLWithPath:shellUtilityPath];
+                [self foundShellUtilityAtURL:shellUtilityUrl];
+            }
+        }
+    }
+}
+
+- (void)foundShellUtilityAtURL:(NSURL *)url
+{
+    self->shellUtilityURL = url;
+    [self indicateShellUtilityInstalledAt:url];
+}
+
+- (void)indicateShellUtilityInstalledAt:(NSURL *)url
+{
+    self.supportIndicator.textColor = self->installedColor;
+    [self.supportTextField setStringValue:NSLocalizedString(@"Shell utility installed", @"Label stating that shell utility has been installed")];
+    [self.locationTextField setStringValue:url.path];
+    NSFont *installedLocationFont = [NSFont fontWithName:@"Menlo" size:self.locationTextField.font.pointSize];
+    [self.locationTextField setFont:installedLocationFont];
+    [self.installUninstallButton setTitle:NSLocalizedString(@"Uninstall", @"Uninstall shell utility button")];
+    [self.installUninstallButton setAction:@selector(unInstallShellUtility)];
+}
+
+- (void)indicateShellUtilityNotInstalled
+{
+    self.supportIndicator.textColor = self->notInstalledColor;
+    [self.supportTextField setStringValue:NSLocalizedString(@"Shell utility not installed", @"Label stating that shell utility has not been installed")];
+    [self.locationTextField setStringValue:NSLocalizedString(@"<Not installed>", @"Displayed instead of path when shell utility has not been installed")];
+    NSFont *notInstalledFont = [[NSFontManager sharedFontManager] convertFont:
+                                [NSFont systemFontOfSize:self.locationTextField.font.pointSize]
+                                toHaveTrait:NSFontItalicTrait];
+    [self.locationTextField setFont:notInstalledFont];
+    [self.installUninstallButton setTitle:NSLocalizedString(@"Install", @"Install shell utility button")];
+    [self.installUninstallButton setAction:@selector(installShellUtility)];
+}
+
+- (void)installShellUtility
+{
+    // URL for macdown utility in .app bundle
+    NSBundle *bundle = [NSBundle mainBundle];
+    NSURL *utilityBundleUrl = [[bundle sharedSupportURL] URLByAppendingPathComponent:@"bin/macdown"];
+    
+    if ([[NSFileManager defaultManager] fileExistsAtPath:[utilityBundleUrl path]])
+    {
+        NSString *installPath = @"/usr/local/bin/macdown";
+        
+        NSError *copyError;
+        [[NSFileManager defaultManager] copyItemAtPath:utilityBundleUrl.path toPath:installPath error:&copyError];
```

Comment:
I just realised Homebrew actually *links* the executable instead of copying. Would it be a better idea here?